src/server_manager/cloud/gcp_api.ts

```diff
@@ -43,6 +43,62 @@ export type Instance = Readonly<{
   }>;
 }>;
 
+const GCE_V1_API = 'https://compute.googleapis.com/compute/v1';
+
+function projectLink(projectId: string): string {
+  return `${GCE_V1_API}/projects/${projectId}`;
+}
+
+export interface RegionScope {
```

Comment:
For consistency with InstanceLocator:
RegionScope -> RegionLocator
ZoneScope -> ZoneLocator

Reply:
Done